### ISSUE TITLE
Relative energy calculation uses DFT minimum as reference instead of ReaxFF minimum

### ISSUE THREAD

**Issue by un123-ctrl** [cagrikymk/JAX-ReaxFF#26]
Dear author,
    Thank you very much for sharing the code. 
     In JAX-ReaxFF's evaluation/training mode, the relative energies of ReaxFF are computed using the DFT energy minimum from trainset.in as the reference point, rather than using the energy minimum obtained under the ReaxFF force field itself.this way of calculating relative energies reasonable?
     Code location: optimizer.py:274
     **energy_preds = jnp.sum(all_energy[energy_items.sys_inds] * energy_items.multip, axis=1)**
     Version: latest on GitHub
